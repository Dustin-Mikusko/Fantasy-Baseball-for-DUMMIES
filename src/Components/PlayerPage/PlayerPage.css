.player-card {
  color: #041F42;
  border: 7px ridge #041F42;
  width: 25%;
  height: 360px;
  font-size: 1.4em;
  padding: 15px;
  margin: auto;
  margin-top: 40px;
  position: relative;
}

.player-name {
  border-bottom: 2px solid #041F42;
  text-align: center;
  margin-top: 5px;
  font-family: Astros;
}

.tag {
  font-weight: bold;
}

.bats {
  margin-bottom: 10px;
}

.view-stats-btn {
  text-decoration: none;
  color: white;
  background-color: #041F42;
  width: 150px;
  margin: auto;
  margin-top: 45px;
  padding: 10px;
  border-radius: 3px;
  position: absolute;
  text-align: center;
  bottom: 10px;
  left: 100px;
}

.view-stats-btn:hover {
  background-color: rgb(6, 71, 155);
}

.view-stats-btn:active {
  transform: scale(.97);
}
