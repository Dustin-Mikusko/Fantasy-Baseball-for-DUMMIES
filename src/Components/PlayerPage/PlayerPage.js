import React from 'react';
import './PlayerPage.css';
import { Link } from 'react-router-dom';

export const PlayerPage = ({ player }) => {
  return (
    <>
      <h1>{player.name}</h1>
      <p>Team: {player.team}</p>
      <p>Position: {player.position}</p>
      <p>Height: {player.feet}' {player.inches}"</p>
      <p>Weight: {player.weight}lbs.</p>
      <p>Bats/Throws: {player.bats}/{player.throws}</p>
      <Link to={`player/${player.id}`}>VIEW STATS</Link>
    </>
  )
};

export default PlayerPage;

