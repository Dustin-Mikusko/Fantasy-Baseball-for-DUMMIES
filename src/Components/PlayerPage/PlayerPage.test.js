import React from 'react';
import PlayerPage from './PlayerPage';
import { shallow } from 'enzyme';

describe('PlayerPage', () => {
  it('should match the snapshot with correct data', () => {
    const mockPlayer =  {
      name: 'Jose Altuve',
      team: 'Houston Astros',
      position: '2B',
      inches: '6',
      feet: '5',
      weight: '165',
      bats: 'R',
      throws: 'R',
      id: '234'
    };
    const wrapper = shallow(<PlayerPage 
      player={mockPlayer}
    />);
      
    expect(wrapper).toMatchSnapshot();
  });
})
