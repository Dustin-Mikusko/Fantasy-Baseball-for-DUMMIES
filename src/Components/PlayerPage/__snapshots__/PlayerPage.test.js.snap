// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerPage should match the snapshot with correct data 1`] = `ShallowWrapper {}`;
